### PR TITLE
[REF] mail: `WelcomeView`, review and simplify scss

### PR DESCRIPTION
**Pull request by Brieuc-brd**
Part of the overall v16 SCSS optimization/restyle, task-2704984

task-2852562

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by Brieuc-brd** [comment]
Hello @alexkuhn 👋 
This PR is ready for review.

cc. @stefanorigano

File: addons/mail/static/src/components/welcome_view/welcome_view.xml [hunk 1]
```diff
@@ -13,13 +13,15 @@
...
                         </t>
                         <t t-if="messaging.currentUser">
-                            <p class="o_WelcomeView_loggedAsStatus">Logged as <span t-esc="messaging.currentUser.nameOrDisplayName"/></p>
+                            <p class="o_WelcomeView_loggedAsStatus fs-4">Logged as <span t-esc="messaging.currentUser.nameOrDisplayName"/></p>
                         </t>
-                        <button class="o_WelcomeView_joinButton btn btn-success btn-lg align-self-end p-0 rounded-circle shadow fa fa-phone" title="Join Channel" t-att-disabled="welcomeView.isJoinButtonDisabled" t-on-click="welcomeView.onClickJoinButton"/>
+                        <button class="o_WelcomeView_joinButton btn btn-success fa-stack align-self-end p-0 rounded-circle fs-1 shadow" title="Join Channel" t-att-disabled="welcomeView.isJoinButtonDisabled" t-on-click="welcomeView.onClickJoinButton">
+                            <i class="fa fa-phone fa-inverse fa-stack"/>
```
Comment (alexkuhn) [inline addons/mail/static/src/components/welcome_view/welcome_view.xml:23]: I don't see any change with or without `fa-inverse` (already proper color).

**Review by alexkuhn — COMMENTED** [review]
LGTM otherwise

File: addons/mail/static/src/components/welcome_view/welcome_view.xml (hunk 1 above)
Comment (Brieuc-brd) [reply addons/mail/static/src/components/welcome_view/welcome_view.xml]: Updated 👍

**Review by alexkuhn — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**